modules/flowable-event-registry-integration-test/src/test/java/org/flowable/eventregistry/integrationtest/ProcessWithDynamicChannelProcessorTestConfiguration.java

```diff
@@ -0,0 +1,193 @@
+package org.flowable.eventregistry.integrationtest;
```

Comment:
Please include the license header; thanks.

Reply:
fixed